photutils/psf/building/core.py

```diff
@@ -0,0 +1,109 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+Core functionalities for building a PSF.
+"""
+from __future__ import absolute_import, division, print_function
+
+__all__ = ['PSFBuilder', 'build_psf']
+
+
+# TODO: Implement based on API from Erik Tollerud.
```

Comment:
@eteq , does this reasonably represent your [API concept](https://github.com/spacetelescope/sprint_notebooks/blob/master/build_PSF_API_concept.ipynb)?